language/tools/move-mv-llvm-compiler/src/stackless/llvm.rs

```diff
@@ -284,6 +288,26 @@ impl Module {
         }
     }
 
+    #[allow(clippy::not_unsafe_ptr_arg_deref)]
+    pub fn add_type_attribute(
+        &self,
+        func: Function,
+        idx: llvm_sys::LLVMAttributeIndex,
+        name: &str,
+        ty: LLVMTypeRef,
```

Comment:
This should take struct `Type` instead of `LLVMTypeRef`. When that is done, the `clippy::not_unsafe_ptr_arg_deref` can be removed.